pages/events/_id/edit.vue

```diff
@@ -238,8 +238,19 @@ export default {
           },
           {
             name: 'price',
-            placeholder: 'Examples: 10EUR, 5USD, FREE',
-            description: 'Examples: 10EUR, 5USD, FREE',
+            type: 'price',
+            placeholder: 'Examples: 10, 5, FREE',
+            description: 'Examples: 10, 5, FREE',
+            defaultPrice: {
+              amount: 0,
+              currency: 'EUR',
+            },
+            options: [
+              { name: 'EUR' },
+              { name: 'USD' },
```

Comment:
if then, this has just 4 currency symbols?

Reply:
Done.👍 Now It is a list with currency abbreviation and symbol.